translations/en.json

```diff
@@ -1754,8 +1755,9 @@
     "contact-request-pending": "Pending...",
     "removed-from-contacts": "Removed from contacts",
     "mutual-contact-requests": "Mutual contact requests",
-    "pending": "Pending",
     "negative": "Negative",
     "positive": "Positive",
-    "public": "Public"
+    "public": "Public",
+    "app-connections": "App connections",
```

Comment:
app-connections duplicated

Reply:
removed duplicate key, again sorry for missing this out after rebasing develop